index.html

```diff
@@ -92,7 +92,7 @@ <h1>Slideshow Test</h1>
       </div>
       <div class="dcf-grid">
         <div class="dcf-col-100% dcf-col-33%-start@sm dcf-col-50%-start@lg">
-          <div class="dcf-slideshow dcf-slideshow-nojs" data-fade role="region" aria-label="slideshow" tabindex="0">
+          <div class="dcf-slideshow" data-fade role="region" aria-label="slideshow" tabindex="0">
```

Comment:
Not really a big deal but the `aria-label` for each slide show should be unique.  It would cause axe error in web audit.